build/Targets/Settings.props

```diff
@@ -10,9 +10,9 @@
     <DeveloperBuild>false</DeveloperBuild>
     <DeveloperBuild Condition="'$(OfficialBuild)' != 'true' AND '$(JenkinsBuild)' != 'true'">true</DeveloperBuild>
 
-    <ProjectDir>$(MSBuildThisFileDirectory)..\..\</ProjectDir>
-    <NuGetToolPath>$(ProjectDir)nuget.exe</NuGetToolPath>
-    <ToolsetPackagesDir>$(ProjectDir)build\ToolsetPackages\</ToolsetPackagesDir>
+    <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)..\..\'))</RepoRoot>
```

Comment:
If you grep the default installed set of targets, ProjectDir is set to MSBuildProjectDirectory but not consumed anywhere. I think this exists so people who wanted to use pre/post actions had the variable defined as an environment variable so they could type a shorter name. Deletion here is correct.